dags/multipod/maxtext_stable.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A DAG to run MaxText tests with stable version."""
+
+
+import datetime
+from airflow import models
+from dags import composer_env
+from dags.vm_resource import TpuVersion, Zone
+from dags.multipod.configs import maxtext_gce_config
+
+
+# Run once a day at 4 am UTC (8 pm PST)
+SCHEDULED_TIME = "0 4 * * *" if composer_env.is_prod_env() else None
+
+
+with models.DAG(
+    dag_id="maxtext_stable",
+    schedule=SCHEDULED_TIME,
+    tags=["multipod_team", "maxtext", "stable"],
+    start_date=datetime.datetime(2024, 1, 19),
+    catchup=False,
+) as dag:
+  test_name_prefix = "maxtext-stable"
+
+  maxtext_stable_llama2_v4_8 = maxtext_gce_config.get_maxtext_stable_config(
```

Comment:
We really want these to run on both nightly and stable if possible. Generally, when folks write tests that is how I want them run so we should try to make that super easy.